scripts/src/javatests/org/oppia/android/scripts/docs/KdocValidityCheckTest.kt

```diff
@@ -0,0 +1,875 @@
+package org.oppia.android.scripts.docs
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.oppia.android.scripts.proto.KdocValidityExemptions
+import org.oppia.android.testing.assertThrows
+import java.io.ByteArrayOutputStream
+import java.io.File
+import java.io.PrintStream
+
+/** Tests for [KdocValidityCheck]. */
+class KdocValidityCheckTest {
```

Comment:
```suggestion
class KDocValidityCheckTest {
```

Reply:
Ah! this has been discussed previously, and it was decided to name it as "Kdoc", as "Kdoc" is a single entity in itself.
Ditto for the script name.

Reply:
For context, see also: https://developer.android.com/kotlin/style-guide#camel_case for why it should be 'Kdoc' and not 'KDoc' here.